installer/templates/phx_new/lib/app_name/web.ex

```diff
@@ -0,0 +1,76 @@
+defmodule <%= application_module %>.Web do
+  @moduledoc """
+  A module that keeps using definitions for controllers,
+  views and so on.
```

Comment:
@chrismccord @josevalim Yup, `MyApp.Web, :context` is def. less than ideal. Perhaps on HexWeb we'll end up having non-Web context module to use (to avoid copy pasting aliases/imports) or no module at all; I guess we'll know once we start moving code over to `lib/` from web/models. Thanks for your replies! cc @ericmj